From 81c4c35df222e99f57b694932743cbbba4bbff4e Mon Sep 17 00:00:00 2001
From: Pardha Kesapragada <pkesapragada@habana.ai>
Date: Tue, 16 Mar 2021 18:55:17 +0200
Subject: [PATCH] [SW-0] gaudi2-arm: increase max preboot size

The below commit has increased the prebootsize for
all platforms except gaudi2. Hence make the similar
change in this platform as well, to avoid
compile error.

[SW-34382] preboot: gaudi: increase sram

Change-Id: Ieb69446ed9bb71d97d0cf9b22726874bf9f9a3c5
Signed-off-by: Pardha Kesapragada <pkesapragada@habana.ai>
---

diff --git a/gaudi2_arm/common.h b/gaudi2_arm/common.h
index 1243ae8..20fedfe 100644
--- a/gaudi2_arm/common.h
+++ b/gaudi2_arm/common.h
@@ -14,7 +14,7 @@
 #define HL_SPI_LEGACY_PREBOOT_OFF	0xC0000
 
 #define PPBOOT_SIZE			0x4000		/* 16KB */
-#define PREBOOT_SIZE			0xC000		/* 48KB */
+#define PREBOOT_SIZE			0x10000		/* 64KB */
 #define SRAM_SIZE			0x3000000	/* 48MB */
 #define PPBOOT_TEXT_BASE		0x7FFD000000
 #define PREBOOT_TEXT_BASE		(PPBOOT_TEXT_BASE + PPBOOT_SIZE)
